astro-hello-world/README.md

```diff
@@ -0,0 +1,10 @@
+# astro-hello-world
+
+https://docs.astro.build/en/install/manual/
+
+run
+
+```sh
+$ yarn astro dev
```

Comment:
if you are using `npm-scripts` to set up, why not just use `yarn dev` ?

Reply:
I wrote what I ran before setting npm-scripts.
No problem with `yarn dev`!
Thank you for your feedback!